src/node/wrapper.ts

```diff
@@ -321,6 +321,7 @@ export class ParentProcess extends Process {
       env: {
         ...process.env,
         CODE_SERVER_PARENT_PID: process.pid.toString(),
+        CS_NODE_EXEC_PATH: path.dirname(process.execPath) + ":" + process.env.PATH,
```

Comment:
This will need to be the path to just Node.

```suggestion
        CS_NODE_EXEC_PATH: process.execPath,
```